scripts/cluster-install/postgresql_config.yaml

```diff
@@ -0,0 +1,20 @@
+postgresql_server:
+  cert_path: /home/centos/install_cluster/certs/<db_instance_name>_cert.pem
+  key_path: /home/centos/install_cluster/certs/<db_instance_name>_key.pem
+  ca_path: /home/centos/install_cluster/certs/ca.pem
+
+  postgres_password: 'somesecretvalue123!'
+  cluster:
+    nodes:
```

Comment:
Shouldn't this be an empty list initially, or does yaml really parse this successfully?

Reply:
It parses it successfully but I added an empty list for readability.